qualtran/surface_code/physical_parameters.py

```diff
@@ -0,0 +1,48 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import attr
+from attrs import frozen
+
+
+@frozen
+class PhysicalParameters:
+    """PhysicalParameters contains physical properties of a quantum computer.
```

Comment:
generally you don't have to re-state the name of the class in the docstring

```suggestion
    """The physical properties of a quantum computer.
```